07-conclusion.Rmd

```diff
@@ -1,27 +1,11 @@
 # Concluding Remarks {#conclusion}
 
-I hope that this book provides a nice gateway into understanding how statisticians, 
-data scientists, and many other professionals use RStudio and R Markdown to simplify
-their analyses and to ensure that their reports are computationally reproducible.
-Learning R is nowhere near as intimidating as it used to be and more and more
-industries are shifting towards free open-source tools like R and RStudio.  R Markdown
-provides an excellent way to document your analyses and share it with others in a
-variety of formats.
+My hope is that this book provides a nice introduction into understanding how statisticians, data scientists, and many other professionals use RStudio and R Markdown to simplify their analyses and ensure that their reports are computationally reproducible. Learning R is not nearly as intimidating as it once was, and more and more industries are shifting towards free open-source tools like R and RStudio. R Markdown provides an excellent way to document your analyses and share it with others in a variety of formats.
 
-Of course, this book is just the tip of the iceberg in terms of showing you what R
-can really do.  If you'd like to learn more I encourage you to check out Albert Kim
-and I's online, free, open-source book on using modern data analysis techniques
-and visualization with R, RStudio, and R Markdown.  More details on that project
-will be coming soon.
+Of course, this book is just the tip of the iceberg in terms of showing you what R can really do. If you'd like to learn more, I encourage you to check out [Modern Dive](https://ismayc.github.io/moderndiver-book/), a free, online, open-source book Albert Kim and I wrote on using modern data analysis techniques and visualization with R, RStudio, and R Markdown.
```

Comment:
@pkq Can you switch this to https://www.moderndive.com instead of my GitHub?

Reply:
Sure. FYI, when I click that link in Firefox, I get a "Your connection is not secure" error. Should it be http://www.moderndive.com?

Reply:
Sigh, yes. I need to get that switched over to Netlify one of these days to get it secure.

Reply:
On it!